### PR TITLE
Show reduced price in ProductItemContractContainer

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Show reduced price for trial ProductItem

Screenshot 2023-11-09 at 17 29 29

## Justify why they are needed
Show same discount as in PriceBreakdown

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3538** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3538?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3538?utm_source=stack-comment).

File: apps/store/src/features/carDealership/ProductItemContractContainer.tsx [hunk 1]
```diff
@@ -17,6 +18,15 @@ export const ProductItemContractContainerCar = ({ contract }: Props) => {
     value: item.displayValue,
   }))
 
+  // Only show crossed over amount if default offer is more expensive than the trial
+  const showCrossedOverAmount =
+    crossedOverAmount !== undefined && crossedOverAmount.amount > contract.premium.amount
+
+  const productPrice = {
```
Comment (alebedev) [inline apps/store/src/features/carDealership/ProductItemContractContainer.tsx:25]: Would this be more readable as single if/else vs two related ternaries?
```
let productPrice
if (showCrossedOverAmount) {
  productPrice = ...
} else {
  productPrice = ...
}

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/features/carDealership/ProductItemContractContainer.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/features/carDealership/ProductItemContractContainer.tsx:25]: I like it, let's change it 🙌 @alebedev

**Comment by gustaveen** [comment]
### Merge activity

* **Nov 10, 4:52 AM**: @gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3538).

**Pull request merged**